pages/content/amp-dev/documentation/courses/beginning-course/thinking-in-components@pt_BR.md

```diff
@@ -0,0 +1,265 @@
+| `<amp-img src="IMG-URL" layout="responsive" width="640" height="480"></amp-img>`             | {{ image('/static/img/courses/beginner/image14.png', 311, 550) }} |
+| `<amp-twitter width="486" height="657" layout="responsive" data-tweetid="ID"></amp-twitter>` | {{ image('/static/img/courses/beginner/image19.png', 311, 550) }} |
+| `<amp-youtube data-videoid="ID" layout="responsive" width="480" height="270"></amp-youtube>` | {{ image('/static/img/courses/beginner/image15.png', 311, 550) }} |
+
+O objetivo ao criar seus sites AMP é usar componentes AMP sempre que possível. Os componentes maximizam os benefícios de desempenho da construção com AMP, porque você não precisa criar algo que já existe, aproveitando assim o trabalho dos autores da biblioteca AMP.
+
+Quase todos os componentes AMP são executados por pelo menos algum JavaScript. Para alguns componentes AMP (como `<amp-img>`), o JavaScript é construído diretamente no script do runtime do AMP que você incluiu na parte superior da sua página no boilerplate. Para a maioria dos componentes AMP, você precisará incluir uma tag de script separada. E há um bom motivo para isso: você só inclui os scripts que realmente usa em seu site. Então, os usuários só precisam baixar o código necessário para navegar em sua página. Menos código para baixar significa que seu site carregará mais rápido!
+
+## Exercício 3: Nosso primeiro componente - `<amp-img>`
+
+A maioria das tags HTML pode ser usada diretamente em AMP, mas certas tags, como a tag `<img>`, devem ser substituídas por componentes AMP equivalentes. Esses componentes incorporam as melhores práticas integradas em acessibilidade, capacidade de resposta e desempenho.
+
+Por exemplo, no caso de `<amp-img>`, AMP exige que especifiquemos as dimensões da imagem. O AMP precisa entender o layout da página antes do carregamento dos recursos (como imagens). Isso melhora a experiência do usuário quando sua página está carregando, mas antes que os ativos de imagem tenham sido baixados. Quando as imagens são baixadas, elas podem ser inseridas na página sem que nenhum conteúdo existente na página se mova. Isso dá ao runtime do AMP margem para decidir quando carregar ativos de imagem com base nos recursos do dispositivo do usuário e na conexão com a Internet.
+
+[tip type="note"]
+**Note**: Se você quiser saber mais sobre as otimizações automáticas de AMP, leia sobre o lazy-loading em AMP no [apêndice]({{g.doc('/content/amp-dev/documentation/courses/beginning-course/appendix.md', locale=doc.locale).url.path}}).
+[/tip]
+
+Para usar o componente e resolver o erro de validação `<amp-img>` anterior, substitua a tag img existente em nossa página pelo equivalente de AMP.
+Dica: em vez de `<img…>`, escreva `<amp-img…>` e atribua dimensões fixas à imagem. Dê à imagem uma `largura` de` 640` e uma `altura` de` 480`.
```

Comment:
For some reason, when you place these sentences (starting with "Dica:") on a new line, it confuses the parser. I don't know why that is... but if you remove the newline before "Dica:", it does fix the problem!

Reply:
Done!